api/spec/v1.yaml

```diff
@@ -1370,6 +1402,32 @@ components:
       description: |
         A ParaTime block.
 
+    RuntimeTransactionList:
+      type: object
+      properties:
+        transactions:
+          type: array
+          items:
+            $ref: '#/components/schemas/RuntimeTransaction'
+      description: |
+        A list of consensus transactions.
+
+    RuntimeTransaction:
+      type: object
+      properties:
+        round:
+          type: integer
+          format: int64
+          description: The block round at which this transaction was executed.
+          example: *runtime_block_round_1
+        hash:
+          type: string
+          description: The Oasis cryptographic hash of this transaction's encoding.
+          example: *tx_hash_1
+        # todo: other fields
```

Comment:
Can you document `To` and `Signer`? I see lots of potential for confusion there. First, the type of address (oasis-sdk address vs underlying eth address). Second, the interpretation of these fields for `ConsensusDeposit` and `ConsensusWithdraw` (and probably `EvmCreate`). I'm actually not clear on that myself. For `ConsensusDeposit`, `ConsensusWithdraw`, if both addresses are sdk addresses, how do we know the affected consensus address?

Reply:
oh yeah this is not filled in at all yet :facepalm:

Reply:
filled